Content/src/Client/Client.fs

```diff
@@ -98,19 +107,22 @@ let initialCounter () = Fetch.fetchAs<Counter> "/api/init"
 // defines the initial state
 let init () : Model =
     { Counter = None }
+#elseif (bridge)
+let init () : Model * Cmd<Msg> =
+    { Counter = None }, Cmd.none
 #else
 // defines the initial state and initial command (= side-effect) of the application
 let init () : Model * Cmd<Msg> =
     let initialModel = { Counter = None }
     let loadCountCmd =
 #endif
-#if (!reaction && remoting)
+#if (!reaction && remoting && !bridge)
```

Comment:
those 3 conditions got bit complex, can we refactor those to make it cleaner? Maybe use nested `#if` statements?

Reply:
I'll give it a try